roles/traefik-proxmox/tasks/main.yml

```diff
@@ -72,6 +72,10 @@
                 published: 443
                 protocol: tcp
                 mode: host
+              - target: 8082
+                published: 8082
+                protocol: tcp
+                mode: host
```

Comment:
This will publicly expose the metrics endpoint. Is it protected otherwise somehow that ive missed?

Reply:
I'd need to double check with @ocanty but from WAN only ports 80 and 443 get forwarded to Traefik

![](https://snipboard.io/RBErs8.jpg)

https://github.com/UCCNetsoc/NaC/blob/master/provision-router.yml#L68

Reply:
this is correct